config.test.env

```diff
@@ -1,3 +1,4 @@
+DEBUG=False
```

Comment:
Could you add this to `config.tmpl.env` too please?

Reply:
Of course, good pick up.